src/attr/incomparable.rs

```diff
@@ -0,0 +1,52 @@
+//! Attribute parsing for the `default` option.
+
+use proc_macro2::Span;
+use syn::{spanned::Spanned, Meta, Result};
+
+use crate::{attr::DeriveTrait, DeriveWhere, Error};
+
+/// Stores if this variant should be the default when implementing
+/// [`Default`](std::default::Default).
+#[derive(Clone, Copy, Default)]
+#[cfg_attr(test, derive(Debug))]
+pub struct Incomparable(pub Option<Span>);
+
+impl Incomparable {
+	/// Token used for the `default` option.
+	pub const INCOMPARABLE: &'static str = "incomparable";
+
+	/// Adds a [`Meta`] to this [`Default`](self).
```

Comment:
```suggestion
	/// Adds a [`Meta`] to this [`Incomparable`].
```
Could you also fix the typo in `default.rs`: ``[`Default`](self)`` should be ``[`Default`](Self)``.